packages/mechanic-template/functions/index.js

```diff
@@ -93,12 +93,41 @@ const runP5 = (functionName, values, isPreview) => {
   );
 };
 
+// SVG
+// -------------------------------------------------------------------------
+
+const runSVG = (functionName, values, isPreview) => {
+  root.innerHTML = "";
+  const func = functions[functionName];
+  const mechanic = new Mechanic(func.params, func.settings, values);
+
+  const onFrame = el => {
+    root.innerHTML = "";
+    root.appendChild(el);
+    if (!isPreview) {
+      mechanic.frame(el);
+    }
+  };
```

Comment:
I knew this problem around appending a new SVG element on every frame would arise, and I'm afraid it will result in slow preview rendering times. We could consider using something along the lines of [virtual-dom](https://www.npmjs.com/package/virtual-dom) so we don't need to insert a new element on every frame.